### PR TITLE
Logo for google search added

### LINKED ISSUE #3174: Add logo for search results
Currently, a search for mozilla foundation yields the awesome but extinct dinosaur. 

Seems like we can update this via meta info on our site. Info here: https://developers.google.com/search/docs/data-types/logo

cc @kristinashu @alanmoo 

image

### PR DESCRIPTION
**Pull request by Stark-Pramod**
fixes #3174 
Related PRs/issues #
Added the markup in network-api/networkapi/templates/pages/base.html.

### CODE REVIEW

**Comment by Stark-Pramod** [comment]
@alanmoo, please review the Changes

**Comment by Stark-Pramod** [comment]
@alanmoo yes I took the reference from [here](https://searchengineland.com/schema-markup-structured-data-seo-opportunities-site-type-231077).
location of the logo is changed.
Please guide me if I am wrong as I told you I am new to open source.

File: network-api/networkapi/templates/pages/base.html [hunk 1]
```diff
@@ -18,7 +18,15 @@
     <title>{% block pageTitle %}Mozilla Foundation - {% if page.specifics.seo_title %}{{ page.specifics.seo_title }}{% else %}{{ page.title }}{% endif %}{% endblock %}</title>
   </head>
   <body class="{% block bodyclass %}{% endblock %}" id="view-{% block bodyID %}{% endblock %}">
-
+   <script type="application/ld+json">
+   {
+     "@context" : "http://schema.org",
+     "@type" : "Organization",
+     "legalName" : "Mozilla Foundation",
+     "url" : "https://foundation.mozilla.org/en/",
```
Comment (alanmoo) [inline network-api/networkapi/templates/pages/base.html]:
```suggestion
     "url" : "https://foundation.mozilla.org/",
```

**Comment by Stark-Pramod** [comment]
@alanmoo sure.

**Comment by alanmoo** [comment]
Looking good, I'm just going to let the review app spin up and CI run.

**Comment by Stark-Pramod** [comment]
@alanmoo ok and thanks.

**Comment by alanmoo** [comment]
@patjouk @pomax any idea why Percy dooesn't seem to be firing?

**Comment by patjouk** [comment]
@alanmoo Travis [doesn't want to leak any secure env var](https://docs.travis-ci.com/user/pull-requests#pull-requests-and-security-restrictions). Since the percy run depends of a secure token, external contributors won't be able to run it.

**Comment by Stark-Pramod** [comment]
@patjouk whats the solution?

**Comment by alanmoo** [comment]
In this case I'm fine leveraging admin powers to force this in without Percy, as this doesn't affect the rendered page. Does that sound reasonable, @patjouk? (Obviously this is something we'd have to figure out if we got more contributions)

**Comment by patjouk** [comment]
works for me

**Comment by alanmoo** [comment]
Thanks, @Stark-Pramod! Hopefully this does the trick!

**Comment by Stark-Pramod** [comment]
@alanmoo hope so.

**Pull request merged**